Lecture-4/Task2/Task2/Application.java

```diff
@@ -0,0 +1,37 @@
+package Task2;
+
+import java.util.Scanner;
+
+public class Application {
+
+    public static void main(String[] args) {
+        System.out.println("type the day number: ");
+        Scanner scanner = new Scanner(System.in);
+        int dayNumber = scanner.nextInt();
```

Comment:
This is ok approach.
Another good option for this task is to use Java 8 Date Time API:
`System.out.println(DayOfWeek.of(2))`